Signal/src/call/PeerConnectionClient.swift

```diff
@@ -1092,6 +1110,90 @@ class HardenedRTCSessionDescription {
     }
 }
 
+protocol VideoCaptureSettingsDelegate: class {
+    var videoWidth: Int32 { get }
+    var videoHeight: Int32 { get }
+}
+
+class VideoCaptureController {
+
+    let capturer: RTCCameraVideoCapturer
+    weak var settingsDelegate: VideoCaptureSettingsDelegate?
+    var isUsingFrontCamera: Bool = true
```

Comment:
I'm slightly concerned about thread safety.  We seem to access `VideoCaptureController` on the main thread and the "signaling queue".  `isUsingFrontCamera` for example is mutated and accessed on both, right?  Should we make the properties of this class private and synchronize access to them?

Reply:
Good point - I've serialized read/write to the affected methods.